EventsExpress/Controllers/UnitOfMeasuringController.cs

```diff
@@ -4,14 +4,15 @@
 using AutoMapper;
 using EventsExpress.Core.DTOs;
 using EventsExpress.Core.IServices;
+using EventsExpress.ModelBinders;
 using EventsExpress.ViewModels;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
 namespace EventsExpress.Controllers
 {
     [Route("api/[controller]")]
-    [Authorize]
+    [Authorize(Roles = "Admin")]
```

Comment:
Admin should be only for create/update/delete. The 'All' action should be available for all to be able to use on event forms.